web3_proxy/src/rpcs/many.rs

```diff
@@ -308,8 +320,43 @@ impl Web3Rpcs {
             }
         }
 
+        // <<<<<<< HEAD
         Ok(())
     }
+    // =======
+    //         // TODO: max_capacity and time_to_idle from config
+    //         // all block hashes are the same size, so no need for weigher
+    //         let block_hashes = Cache::builder()
+    //             .time_to_idle(Duration::from_secs(600))
+    //             .max_capacity(10_000)
+    //             .build_with_hasher(hashbrown::hash_map::DefaultHashBuilder::default());
+    //         // all block numbers are the same size, so no need for weigher
+    //         let block_numbers = Cache::builder()
+    //             .time_to_idle(Duration::from_secs(600))
+    //             .max_capacity(10_000)
+    //             .build_with_hasher(hashbrown::hash_map::DefaultHashBuilder::default());
+    //
+    //         let (watch_consensus_connections_sender, consensus_connections_watcher) =
+    //             watch::channel(Default::default());
+    //
+    //         let watch_consensus_head_receiver =
+    //             watch_consensus_head_sender.as_ref().map(|x| x.subscribe());
+    //
+    //         let connections = Arc::new(Self {
+    //             by_name: connections,
+    //             watch_consensus_rpcs_sender: watch_consensus_connections_sender,
+    //             watch_consensus_head_receiver,
+    //             pending_transactions,
+    //             block_hashes,
+    //             block_numbers,
+    //             min_sum_soft_limit,
+    //             min_head_rpcs,
+    //             max_block_age,
+    //             max_block_lag,
+    //         });
+    //
+    //         let authorization = Arc::new(Authorization::internal(db_conn.clone())?);
+    // >>>>>>> 77df3fa (stats v2)
```

Comment:
looking at main, this all seems to have moved. i think this stale code can be deleted

Reply:
Indeed, I am removing this